lib/puppet/network/formats.rb

```diff
@@ -30,7 +30,7 @@ def intern(klass, text)
   def intern_multiple(klass, text)
     data = YAML.load(text)
     unless data.respond_to?(:collect)
-      raise Puppet::Network::FormatHandler::FormatError, "Serialized YAML did not contain a collection of instances when calling intern_multiple"
+      raise Puppet::Network::FormatHandler::FormatError, _("Serialized YAML did not contain a collection of instances when calling intern_multiple")
```

Comment:
I'm going to assume that YAML is the same in all languages and the the translators will know to leave it alone.